packages/artifacts/chunks/languages/lioncore-types.ts.txt

```diff
@@ -0,0 +1,116 @@
+// Warning: this file is generated!
+// Modifying it by hand it useless at best, and sabotage at worst.
+
+
+/*
+ * language's metadata:
+ *     name:    LionCore.M3
+ *     version: 2023.1
+ */
+
+
+import {DynamicNode, DynamicINamed as INamed} from "@lionweb/core";
+
+
+export type Annotation = DynamicNode & {
+    classifier: "Annotation";
+    settings: {
+        multiple?: boolean;
+        annotates?: Classifier;
+        extends?: Annotation;
+        implements: ConceptInterface[];
+    };
+};
+
+/** abstract */ export type Classifier = DynamicNode & {
+    classifier: "Annotation" | "Concept" | "ConceptInterface";
```

Comment:
This "seales" the abstract concept, right? Another language could not define its own concept that extends `Classifier`.

Reply:
Correct. I put it in there, because the `shapes` language benefits from it. But I'll only activate that flag for that language then.